operate/client/src/index.scss

```diff
@@ -5,10 +5,7 @@
  * except in compliance with the proprietary license.
  */
 
-@use '@carbon/react' as * with (
-  $font-path: 'https://fonts.camunda.io'
-);
-
+@use '@carbon/react' as *;
```

Comment:
The correct config is like this: https://github.com/carbon-design-system/carbon/blob/main/examples/vite/src/index.scss

Reply:
For Tasklist it's the opposite, it doesn't work if we don't like that. I guess it's because of Vite

We'll have to change this when we finally manage to migrate Operate